backend/src/Server.ts

```diff
@@ -57,7 +57,7 @@ class Server {
                     throw err;
                 } else {
                     CMSAuthService.db = client.db(process.env.DB_NAME as string);
-                    TutorialDbService.db = client.db(process.env.DB_NAME as string);
+                    TutorialUpdateService.db = client.db(process.env.DB_NAME as string);
```

Comment:
This is code duplication. Can you store `client.db(process.env.DB_NAME) as string` in a variable.

Reply:
Sure thing, I'll store it in a field (static).

Reply:
so you'll change that later. okay.

Reply:
If ima change one I basically got change all process.env's :/. And some static classes require the env before it can be initialised so may need package to set them idk, I'll look into it